system_server.te

```diff
@@ -15,3 +15,8 @@ dontaudit system_server hal_audio_default:file write;
 dontaudit system_server appdomain:file write;
 
 allow system_server proc_cmdline:file r_file_perms;
+allow system_server netmgrd_socket:dir search;
+allow system_server netmgrd_socket:sock_file write;
+allow system_server netmgrd:unix_stream_socket connectto;
```

Comment:
Use unix_socket_connect(system_server, netmgrd, netmgrd) instead of these two previous lines.